### PR TITLE
Feat/header

### PR DESCRIPTION
**Pull request by Hudayberdiev2000**

### CODE REVIEW

File: src/components/molecules/Header/Header.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import Header from "./Header";
+
+export const Default = {
+  // args: {
```
Comment (bruno-keiko) [inline src/components/molecules/Header/Header.stories.tsx]: Please remove unnecessary comments

File: src/components/molecules/Header/Header.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  //   image: "icons/highlight-icon.svg",
+  //   title: "100 % Fresh Foods",
+  //   description:
+  //     "Beguiled and demoralized by all get charms pleasure the moments ever so blinded by desire.",
+  //   link: "/",
+  // },
+};
+
+const headerConfig = {
+  title: "Atoms/Header",
```
Comment (bruno-keiko) [inline src/components/molecules/Header/Header.stories.tsx]: Change it to molecules

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
This component doesn't seem to match the design.

File: src/components/molecules/Header/Header.module.scss [hunk 3]
```diff
@@ -0,0 +1,75 @@
+@use "sass:map";
+@import "../../../foundations/sass/all";
+
+.header {
+  box-shadow: 0 5px 5px 0 #0003;
```
Comment (bruno-keiko) [inline src/components/molecules/Header/Header.module.scss]: Could you use a variable in colors?

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Good job! We need minor changes

File: src/components/molecules/Header/Header.module.scss (hunk 3 above)
Comment (Hudayberdiev2000) [reply src/components/molecules/Header/Header.module.scss]: sure.

**Comment by Hudayberdiev2000** [comment]
Has made the change!

**Pull request merged**